about.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Capstone Project 1</title>
+	<link rel="stylesheet" href="style.css">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/lato-font/3.0.0/css/lato-font.min.css">
+</head>
+
+<body>
+	<header class="container">
+		<div class="sub-container1">
+			<ul class="link1">
+				<li class="list-item"><a href="#"><i class="fa fa-facebook"></i></a></li>
+				<li class="list-item"><a href="#"><i class="fa fa-twitter"></i></a></li>
+				<li class="list-item"><a href="#">English</a></li>
+				<li class="list-item"><a href="#">My Page</a></li>
+				<li class="list-item"><a href="#">Logout</a></li>
+			</ul>
+		</div>
+		<div class="sub-container2">
+			<div class="logo">
+				<a href="index.html"><img src="./images/logo2.png" alt=""></a>
+			</div>
+			<ul class="link2">
+				<li class="list-item2 mobile"><a href="index.html">Home</a></li>
+				<li class="list-item2"><a href="about.html">About</a></li>
+				<li class="list-item2"><a href="#">Program</a></li>
+				<li class="list-item2"><a href="#">Join</a></li>
+				<li class="list-item2"><a href="#">Sponsor</a></li>
+				<li class="list-item2"><a href="#">News</a></li>
+			</ul>
+		</div>
+	</header>
+
+	<section class="homepage">
+		<div class="hamburger">
+			<div class="bars"></div>
+		</div>
+		<div class="container2">
+			<h1 class="header10">"Hello!"</h1>
+			<h2 class="header7">Health Awareness on the connection between Diabetes and Obesity Programme 2023</h2>
+			<p class="paragraph7">The Health Awareness Programme brings together a community of Doctors, Nurses,
+				Executives from WHO and many other influencial health personels every two years. This year, we hope to
+				educate and help a lot of individuals in the race to eradicate or reduce the rate of occurence of
+				diabetes mellitus.</p>
```

Comment:
- [ ] Please consider 😃 decreasing the width of the paragraph at the about page  as you see here it is wider than the Behance, so please try to give it width or use any property to make it like the Behance 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/226692960-b2990774-608e-40bd-b8bd-8f3fd1feb99e.png)   |  ![image](https://user-images.githubusercontent.com/83360636/226692888-6bcd8dae-b5e5-4de2-ba23-137edcb36a60.png) |